utils/gyb.py

```diff
@@ -1079,6 +1079,20 @@ def execute_template(
 
 
 def main():
+    """
+    Lint this file.
+    >>> import subprocess, sys
+    >>> script_path = os.path.realpath(__file__)
+    >>> utils_directory = os.path.dirname(script_path)
+    >>> linting = subprocess.Popen([sys.executable,
+    ...                             os.path.join(utils_directory,
+    ...                                         'python-lint'),
+    ...                             script_path])
```

Comment:
This is a technical breaking change (`python-lint` -> `python_lint.py`). If you think it's worth it we can do the same as we do with `gyb` now:
```
#!/usr/bin/env python
import python_lint
python_lint.main()
```